resources/views/editfaq.blade.php

```diff
@@ -0,0 +1,25 @@
+<?php
+use Illuminate\Support\Facades\DB
+?>
+
+<x-layout>
+    <x-slot:title>
+        Edit FAQ Post {{ $post }}
+    </x-slot>
+    <form method="post" action="/editfaqpost">
+        @method('put')
+        @csrf
+        <h1>Vraag: </h1>
+        <input type="text" name="question" value="{{ DB::table('faq')->where('id', $post)->value('question') }}">
```

Comment:
Because of the bad variable name, this line becomes less readable. A better name for the variable would be `faqId`. But you actually should put these kinds of statements in views. It's better to use models instead and let controllers handle the database interactions.